core/mutations/CreateUser.js

```diff
@@ -0,0 +1,11 @@
+import gql from "graphql-tag";
+
+export default gql`
+  mutation signup($username: String, $email: String!, $password: String!) {
```

Comment:
For mutations instead of spreading the input in the arguments let's just use the input for the arguments:
`Signup($input: SignupInput!)`.
This way we don't have to update this file every time fields are added to the input type. The only code that requires update is the one calling the mutation. Type errors in usage will be shown as soon as the schema changes thanks to Graphl codegenerator

Reply:
I'll leave this on hold for now and do it once both core and api are merged since I can't import inputs now, once both are approved I'll do a push with this to wherever is needed